bip-0085.mediawiki

```diff
@@ -51,6 +55,9 @@ For each application that requires its own wallet, a unique private key is deriv
 
 The HMAC-SHA512 function is specified in [http://tools.ietf.org/html/rfc4231 RFC 4231].
 
+Application codes may be arbitrary but are preferably semantic in some way, such as a BIP number or
+ASCII character code sequence.
```

Comment:
Is this needed, and if so, is it added too early? "Application codes" hasn't been defined or referenced yet in the document.

Reply:
It's needed (common question from readers, "where did xyz app code come from." I will move it down to the Application section.